### ISSUE TITLE
Documentation  / Cheatsheet

### ISSUE THREAD

**Issue by whmii** [mrcljx/grunt-liquid#7]
I'm a fan of liquid after getting hooked on it from Mixture. One issue is that documentation for the language has never been really great or accessible. Also the best of it comes from Shopify or the ruby gem which is a mixed bag. I'd be interested in developing some more extensive and approachable documentation for the language a'la http://handlebarsjs.com

@sirlantis , any interest in this?  Doing so may lead to an increased interest in the liquid language.

Also, would it be better if it was associated with the grunt plugin or liquid-node?

**Comment by mrcljx** [comment]
I agree. When I was looking for documentation lately I realised the documentation at Shopify got even worse compared to the last time I visited their documentation site.

I'd prefer to add the documentation to liquid-node (as there is a request for better documentation already, see sirlantis/liquid-node#9) and link there from grunt-liquid and assemble-liquid-engine. Maybe we should put further discussion over there.

Maybe we can ask @jonschlinkert whether we could get the layout/assemble-template of the assemble website, which seems to be a clean and accessible framework for any documentation.

**Comment by whmii** [comment]
Sounds good. I am more of a front-ender/designer so we can put something together. I've been working on converting a boiler plate to a liquid/grunt workflow. I think it would make a good example setup

I seem to be getting hung up on how to configure the Plugin and there's not much to go on.
I'm missing something in the configuration. Any suggestions?
[SuperDry](https://github.com/whmii/SuperDry)
